ui/src/main/webapp/app/query/seed-manager/seed-manager-component.js

```diff
@@ -68,6 +68,8 @@ function SeedManagerController(graph, queryPage, common, types, events) {
 
         if (!message || message === "") {
             message = "No Seeds added. Type in your seeds and click add."
```

Comment:
This error message is shown when the page first loads. Yes, it is useful to prompt users to add seeds, but it is a bit frustrating that an error is shown before you have even done anything. Perhaps you could just change the colour to black and then it would just look like a prompt?